codespell_lib/data/dictionary.txt

```diff
@@ -19765,7 +19765,7 @@ osffset->offset
 osffsets->offsets
 osffsetting->offsetting
 osicllations->oscillations
-ot->to, of, or,
+ot->to, of, or, not,
```

Comment:
possibly also capital "OT" for internet slang "off topic"

Reply:
Although I think it will upset the corrects to itself test...

Reply:
Yeah, we can't do that 🤔